certificate-manager/src/main/java/io/strimzi/certs/CertAndKey.java

```diff
@@ -66,4 +67,15 @@ public String storePassword() {
     public String storePasswordAsBase64String() {
         return Base64.getEncoder().encodeToString(storePassword.getBytes(StandardCharsets.US_ASCII));
     }
+
+    @Override
+    public String toString() {
+        return "CertAndKey{" +
+                "key=" + Arrays.toString(key) +
+                ", cert=" + Arrays.toString(cert) +
+                ", trustStore=" + Arrays.toString(trustStore) +
+                ", keyStore=" + Arrays.toString(keyStore) +
+                ", storePassword='" + storePassword + '\'' +
+                '}';
+    }
```

Comment:
Does this possibly lead to leaking sensitive information in logs, errors and exceptions? Would it be better to have some other method used to give you the String for the checksum (or even directly the checksum itself) to avoid the leakage know or by mistake in the future?

Also, does the hash need to be based on all the fields? The pods don't really use all of them for the broker certificates (Just wodnering -> not sure we really shoudln't use all of them).